lang/cs/Org.Apache.REEF.IO.Tests/TestFilePartitionInputDataSet.cs

```diff
@@ -173,7 +174,7 @@ public void TestWithByteDeserializer()
                         .GetInstance<IInputPartition<IEnumerable<byte>>>();
```

Comment:
This pattern is used everywhere in the tests. If we decide to use T to represent a raw data type, like string or byte, then all those places in the tests need to be updated.